lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -161,16 +116,18 @@ class _WebGlRenderer implements _GlRenderer {
     if (widthInPixels == 0 || heightInPixels == 0) {
       return;
     }
+    String vertexShader = _writeVerticesVertexShader();
+    String fragmentShader = _writeVerticesFragmentShader();
+    print(vertexShader);
```

Comment:
suggestion: Are these from debugging? If not we can maybe print them as 'INFO: <a message>:<value>'

Reply:
Yes debugging. Removed.